sourmash/lca/lca_utils.py

```diff
@@ -317,12 +317,19 @@ def search(self, query, *args, **kwargs):
         return results
 
     def gather(self, query, *args, **kwargs):
+        if not query.minhash:
+            return []
+
         results = []
-        for x in self.find_signatures(query.minhash, 0.0,
+        threshold_bp = kwargs.get('threshold_bp', 0.0)
+        threshold = threshold_bp / (len(query.minhash) * self.scaled)
+        print('xxx', threshold)
```

Comment:
This `print` is really messing the output
```suggestion
```

Reply:
fixed separately